django/db/migrations/questioner.py

```diff
@@ -144,13 +145,13 @@ def ask_not_null_addition(self, field_name, model_name):
         """Adding a NOT NULL field to a model."""
         if not self.dry_run:
             choice = self._choice_input(
-                "You are trying to add a non-nullable field '%s' to %s without a default; "
-                "we can't do that (the database needs something to populate existing rows).\n"
+                "It is impossible to add a non-nullable field '%s' to %s without specifying a default; "
+                " because the database needs something to populate existing rows.\n"
                 "Please select a fix:" % (field_name, model_name),
                 [
-                    ("Provide a one-off default now (will be set on all existing "
-                     "rows with a null value for this column)"),
-                    "Quit, and let me add a default in models.py",
+                    ("Provide a one-off default now which will be set on all existing "
+                     "rows with a null value for this column."),
+                    "Quit so that a default in models.py can be added later.",
```

Comment:
`later` is a bit misleading, since the expectation is that the dev will update models.py (immediately) after quitting so that they can continue to create their migrations.

Reply:
I am considering changing the wording of this sentence to `"Quit and manually define a default value in models.py"`. What do you think about this?

Reply:
Yup, that sounds good too. :)